app/e2e/package.json

```diff
@@ -6,7 +6,7 @@
         "pretest": "yarn g:start-test-db && yarn workspace @app/server prisma-db-push && yarn workspace @app/server prisma-db-seed",
         "test": "yarn pretest && playwright test",
         "test:ci": "dotenv -v CI=1 -- yarn run test",
-        "debug": "playwright test --headed",
+        "debug": "yarn pretest && playwright test --headed",
```

Comment:
Is this a notable change that would need to be documented?

Reply:
I don't think so, it was just a mistake when I originally set it up, it should have had the pretest there.